languages/csharp/exercises/concept/dates/DatesTests.cs

```diff
@@ -12,95 +12,95 @@ public class AppointmentTests
     public void ScheduleDateUsingOnlyNumbers() =>
         Assert.Equal(new DateTime(2019, 07, 25, 13, 45, 0), Appointment.Schedule("7/25/2019 13:45:00"));
 
-    [Fact]
+    [Fact(Skip = "Remove to run test")]
```

Comment:
Remove what to run the test? For complete newbs this may be non-obvious. Perhaps 
```suggestion
    [Fact(Skip = "Remove this line to run this test")]
```

Reply:
That's a possibility, but for what it's worth this is the message we've been using for v2 and I haven't heard of any students getting tripped up on this.

Reply:
I agree that the message can be improved, although I would argue that the suggestion is perhaps not really what we want, as removing the line would also remove the "Fact" part. Furthermore, our current tooling depends on [this specific text](https://github.com/exercism/csharp/blob/master/test.ps1#L55) (it is not great, I know).

For now, I'll merge this as other tracks look for the C# track for guidance. @wolf99 if you think we should still be improving this, could you open a GH issue? That way, we can update it for both the C# and F# tracks.

Reply:
created #1047